docker-compose.yml

```
@@ -0,0 +1,45 @@
+version: '2.4'
+services:
+  redis:
+    image: redis:alpine
+    container_name: pimcore-redis
+
+  db:
+    image: mariadb:10.4
+    container_name: pimcore-mariadb
+    working_dir: /application
+    command: [mysqld, --character-set-server=utf8mb4, --collation-server=utf8mb4_unicode_ci, --innodb-file-format=Barracuda, --innodb-large-prefix=1, --innodb-file-per-table=1]
+    volumes:
+      - pimcore-database:/var/lib/mysql
+    environment:
+      - MYSQL_ROOT_PASSWORD=ROOT
+      - MYSQL_DATABASE=pimcore
+      - MYSQL_USER=pimcore
+      - MYSQL_PASSWORD=pimcore
+
+  adminer:
+    image: adminer
+    ports:
+      - 8080:8080
+
+  php:
+    image: pimcore/pimcore:PHP7.4-apache
+    container_name: pimcore-php
+    volumes:
+      - .:/var/www/html:cached
+    ports:
+     - 80:80
+     - 443:443
+
+  php-debug:
+    image: pimcore/pimcore:PHP7.4-apache-debug
```

`PHP7.4-apache-debug` does not seem to be existing.

```
docker pull pimcore/pimcore:PHP7.4-apache-debug
Error response from daemon: manifest for pimcore/pimcore:PHP7.4-apache-debug not found: manifest unknown: manifest unknown
```
We could downgrade to 7.3 for debug image for now:

```suggestion
    image: pimcore/pimcore:PHP7.3-apache-debug
```

---

Images currently aren't working yet correctly, should work tomorrow (building them takes like forever)

---

I can confirm that PHP7.4-apache-debug is available now:

```
docker pull pimcore/pimcore:PHP7.4-apache-debug
PHP7.4-apache-debug: Pulling from pimcore/pimcore
000eee12ec04: Already exists 
8ae4f9fcfeea: Already exists 
60f22fbbd07a: Already exists 
ccc7a63ad75f: Already exists 
a2427b8dd6e7: Already exists 
91cac3b30184: Already exists 
d6e40015fc10: Already exists 
9858aa646efe: Already exists 
7940985f7eb2: Already exists 
b23f72eebcfb: Already exists 
75bb7b8d192c: Already exists 
7edf943992b0: Already exists 
c8bf9d9d0e11: Already exists 
22e67553f831: Pull complete 
1d659fd986be: Pull complete 
1eec1df56135: Pull complete 
80dbf54decea: Pull complete 
fb6904e26091: Pull complete 
dc00a49875d1: Pull complete 
657c1f64e874: Pull complete 
57740df64ce1: Pull complete 
de0091d9d3b2: Pull complete 
Digest: sha256:ff6aafd19113656049bc7210e148236130a59b9885f0aced540a0f5d443f6f5f
Status: Downloaded newer image for pimcore/pimcore:PHP7.4-apache-debug
docker.io/pimcore/pimcore:PHP7.4-apache-debug
```